app/helpers/application_helper.rb

```diff
@@ -675,6 +675,23 @@ def sort_link_direction(column)
     params[:sort].eql?(column) && params[:direction].eql?("asc") ? "desc" : "asc"
   end
 
+  def search_path(opts = {})
```

Comment:
Maybe this should have a different name so it wouldn't look like an autogenerated path helper fn.